test/fulfil/client_test.rb

```diff
@@ -68,5 +68,14 @@ def test_do_not_retry_when_retry_is_disabled
         end
       end
     end
+
+    def test_put_that_returns_array_wrapped_integer
+      stub_fulfil_put('sale.sale/1/copy', 'sale_sale_copy', '[]')
+
+      client = Fulfil::Client.new
+      order_copy = client.put(model: 'sale.sale', id: 1, endpoint: 'copy', body: [])
```

Comment:
Do we need to send an empty array body to Fulfil?

Reply:
I could be mistaken but without it the test fails and webmock complains.